service.librarywatchdog/changelog.txt

```diff
@@ -0,0 +1,92 @@
+v1.0.0
```

Comment:
please use the `<news>` element in addon xml only for the last(ish) version of the changelog and remove the changelog.txt from the submission if possible. As the changelog is no longer used on the interface, there's no reason for us to ship it from the infrastruture.

Reply:
Done ;)